src/routes/transactions/mappers/common/native-staking.mapper.spec.ts

```diff
@@ -486,9 +485,11 @@ describe('NativeStakingMapper', () => {
           estimatedExitTime: networkStats.estimated_exit_time_seconds,
           estimatedWithdrawalTime:
             networkStats.estimated_withdrawal_time_seconds,
-          value: '96000000000000000000', // 3 public keys in the transaction data => 3 validators * 32 eth
+          value: (
+            +stakes[0].net_claimable_consensus_rewards! +
+            +stakes[1].net_claimable_consensus_rewards!
+          ).toString(),
```

Comment:
These could be summed programatically but this is done to ensure the tests break if any adjustments are made. The same for all other instances.